app/javascript/packs/application.js

```diff
@@ -1,20 +1,25 @@
 /* eslint no-console: 0 */
 
-import Vue from 'vue';
+import Vue from 'vue/dist/vue.esm';
 import VueI18n from 'vue-i18n';
-import App from '../app.vue';
+import Selector from '../selector.vue';
 import Locales from '../locales.js';
 
 Vue.use(VueI18n);
 
 document.addEventListener('DOMContentLoaded', () => {
-  document.getElementById('selector').appendChild(document.createElement('selector'));
-  const app = new Vue({
-    render: h => h(App),
+
+  let renderElement = null;
+  let idElement = '';
```

Comment:
'idElement' is assigned a value but never used            no-unused-vars<br>'idElement' is never reassigned. Use 'const' instead      prefer-const